godtools/App/Share/Common/URLSession/MobileContentApiAuthSession.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  MobileContentApiAuthSession.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 11/10/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import Foundation
+import RequestOperation
+import Combine
+
+class MobileContentApiAuthSession {
```

Comment:
Hey @rachaelblue can you also move this class into Share/Data/MobileContentApiAuthSession/.  Common is reserved for generic classes not related to GodTools.

Reply:
Yep!